test_scripts/API/Restructuring_OnResetTimeout/013_BC_OnResetTimeout_to_DialNumber.lua

```diff
@@ -0,0 +1,61 @@
+------------------------------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+------------------------------------------------------------------------------------------------------------------------
+-- Description: Check SDL does not reset timeout for Mobile app response for a specifc RPC in case
```

Comment:
Please fix typos 
```suggestion
-- Description: Check SDL does not reset timeout for Mobile app response for a specific RPC in case
```
This comment relates to other scripts of this PR